test/Foundatio.Tests/Queue/QueueTestBase.cs

```diff
@@ -28,6 +28,10 @@ public abstract class QueueTestBase : TestWithLoggingBase, IDisposable {
             return null;
         }
 
+        protected virtual Task CloseQueue(IQueue<SimpleWorkItem> queue) {
```

Comment:
I wonder if our other azure queues and rabbit queues are being cleaned up, probably not :\. I'm trying to think in the tests where we create multiple queue instances if they would need to be cleaned up or not. I almost think they wouldn't as they should all have the same queue name but different instance ids.